x-pack/test/api_integration/apis/management/remote_clusters/lib/utils.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+export const wait = (time = 1000) => (
```

Comment:
The retry service also has a `waitFor('description', fn)` function, which re-executes the passed function and waits for it to resolve to a truthy value, up to a timeout.